### PR TITLE
Documentation/tutorials/website improvements

### PR DESCRIPTION
**Pull request by jwmueller**
Mostly miscellaneous improvements in autogluon.text

### CODE REVIEW

**Comment by szha** [comment]
Job PR-968-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-968/1/index.html

**Comment by szha** [comment]
Job PR-968-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-968/2/index.html

**Comment by szha** [comment]
Job PR-968-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-968/3/index.html

**Comment by szha** [comment]
Job PR-968-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-968/4/index.html

File: docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
-We will introduce how to use AutoGluon Tabular to deal with multimodal tabular data that involves text, numeric, and categorical features. This type of data is prevalent in real world applications. In AutoGluon, **raw text data** is considered as a first-class citizen of data tables. AutoGluon Tabular can help you train and combine a diverse set of models including the classical ones like LightGBM/CatBoost and the more recent Pretrained Language Model (PLM) based multimodal network that is introduced in Section ":ref:`sec_textprediction_architecture`" of :ref:`sec_textprediction_multimodal`.
+Here we introduce how to use AutoGluon Tabular to deal with multimodal tabular data that contains text, numeric, and categorical columns. In AutoGluon, **raw text data** is considered as a first-class citizen of data tables. AutoGluon Tabular can help you train and combine a diverse set of models including classical tabular models like LightGBM/RF/CatBoost as well as our pretrained MLP model based multimodal network that is introduced in Section ":ref:`sec_textprediction_architecture`" of :ref:`sec_textprediction_multimodal` (used by AutoGluon's `TextPredictor`).
```
Comment (sxjscience) [inline docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md:5]: "as well as our pretrained MLP model based" --> "as well as our Pretrained Language Model (PLM) based".

Comment (jwmueller) [reply docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md:5]: sorry was supposed to say NLP, fixed now.
I thought we are not calling these language models (and abbreviation PLM is not used anywhere)?

Comment (sxjscience) [reply docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md]: pretrained NLP looks good.

**Review by sxjscience — APPROVED** [review]
The document improvements look good to me.

**Comment by szha** [comment]
Job PR-968-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-968/5/index.html

**Comment by szha** [comment]
Job PR-968-6 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-968/6/index.html

**Pull request merged**